elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/integration/MetaDataStoreIntegrationTest.java

```diff
@@ -40,6 +40,11 @@
 
 public class MetaDataStoreIntegrationTest extends IntegrationTest {
 
+    public MetaDataStoreIntegrationTest() {
+        super();
+        userRoles.addAll(VALIDATOR.getElideSecurityConfig().getRoles());
```

Comment:
Aggregation data store already has a solution to add role checks to the dictionary.  Rather than create a new mechanism here, we can just copy the existing mechanism for this test file.